setup.cfg

```diff
@@ -33,7 +33,7 @@ tests_require =
     pytest
     pytest-django
     isort
-    black
+    black==22.12.0
     flake8
     zgw_consumers
```

Comment:
`tests_require` is leftover from setuptools' `setup.py` direct invocation if I'm not mistaken. I think we can remove it?

Reply:
Possibly, but this is now structured like all our libraries so let's not get distracted by that now and aim for consistency.

I've looked a bit into using pyproject.toml instead and I don't fully understand yet how everything works, while the setup.cfg method is tried and tested and well understood.

If we decide to clean this up, we should do it for all our libraries so that we have a consistent developer experience.

Reply:
Ok, I might do the upgrade on the template and other already running projects then when I have some time!